README.md

```diff
@@ -1,29 +1,38 @@
 
-# Blocking POC 
+# Anonlink Client
 
-A simple end to end demonstrator of blocking for record linkage.
+Client-face API to interact with anonlink system including command line tools and Rest API communication.
+Anonlink system needs the following three components to work together:
 
-Output should be candidate blocks.
+* <a href='https://github.com/data61/clkhash'>clkhash</a>
+* <a href='https://github.com/data61/blocklib'>blocklib</a>
+* <a href='https://github.com/data61/anonlink-entity-service'>anonlink-entity-service</a>
 
-```
-{
-  'records': [clks],
-  'blocks': {
-      'block id 1': [record ids]
-  }
-}
+This package provides an easy to use API to interact with the above packages to complete a record linkage job.
+
+The way to interact with anonlink system is via Command Line Tool `anonlink`. You can hash data containing PII (Personal
+ Identifying Information) locally using `anonlink hash`, generate candidate blocks locally to scale up record linkage 
+ using `anonlink block`, create a record linkage job in entity service with `anonlink create-project` etc.
+
+### Installation
+Currently manual install:
+
+```python3
+pip install git+https://https://github.com/data61/clkhash
 ```
 
-Two methods have been implemented that map to a list of records.
+### Documentation
+https://clkhash.readthedocs.io/en/stable/cli.html
+
+
+### CLI Tool
+After installation, you should have a `anonlink` program in your path. Alternatively, you can use `client.cli`. For
```

Comment:
We can't use the Python module name `client`. Suggest opening an issue.

Reply:
I made an issue #14 for this. Can you provide more details in the issue description? Do you mean `client` is not a recommended module name or you mean you cannot run `client.cli` in your machine?

Reply:
Sure - done.